### PR TITLE
About-Me Section 

### PR DESCRIPTION
**Pull request by fullstop125**
**Portfolio: mobile version**
 
In this section I managed to :  
- Understand how to parse a Figma design to create a UI.
- Use Flexbox to place elements in the page.
- Use CSS Grid to place elements in the page.
- Build a personal portfolio site.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -94,3 +95,175 @@ ul {
 .logo:hover {
   cursor: pointer;
 }
+
+.work-card {
+  padding: 0 1.5rem;
+}
```
Comment (cynthiainga) [inline styles.css]: - [x] Please kindly center your cards :grin: , bellow :point_down: :point_down:  you can see the difference between your design and the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

Your design
Screenshot from 2022-07-01 17-48-50
Figma design
Screenshot from 2022-07-01 17-49-56

File: styles.css [hunk 2]
```diff
@@ -94,3 +95,175 @@ ul {
 .logo:hover {
   cursor: pointer;
 }
+
+.work-card {
+  padding: 0 1.5rem;
```
Comment (cynthiainga) [inline styles.css]: - [x] Kindly also add the border and border radius to your cards :upside_down_face: , your design needs to match as much as possible the Figma design :grin: :grin:

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @fullstop125 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- [ ] Good commit messages :heavy_check_mark: 
- [ ] All linters are passing :heavy_check_mark: 
- [ ] Descriptive PR :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review :point_down: :point_down: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (fullstop125) [reply styles.css]: thank you ..resolved

File: styles.css (hunk 1 above)
Comment (fullstop125) [reply styles.css]: thank you ...resolved

**Review by mirouhml — APPROVED** [review]
Hi @fullstop125, 

Good job implementing the changes requested by the reviewer, your project now is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**